--- php-5.4.19/Zend/zend_language_parser.y	2013-08-19 23:42:41.000000000 -0500
+++ php-5.4.19/Zend/zend_language_parser.y.copy	2013-08-23 08:39:44.776325850 -0500
@@ -47,12 +47,12 @@
 %pure_parser
 %expect 3
 
-%code requires {
+%{
 #ifdef ZTS
 # define YYPARSE_PARAM tsrm_ls
 # define YYLEX_PARAM tsrm_ls
 #endif
-}
+%}
 
 %token END 0 "end of file"
 %left T_INCLUDE T_INCLUDE_ONCE T_EVAL T_REQUIRE T_REQUIRE_ONCE
